src/main/webapp/app/exercises/shared/result/result.component.html

```diff
@@ -18,17 +18,24 @@
                     <span *ngIf="!short" jhiTranslate="artemisApp.result.score">Score</span>
                     {{ round(result.score) }}%,
                 </span>
-                <span *ngIf="hasFeedback">
+                <span *ngIf="resultIsPreliminary(result)">
                     <span class="guided-tour result" (click)="showDetails(result)" [ngbTooltip]="resultTooltip" container="body">
                         {{ resultString }}
                     </span>
                 </span>
-                <span *ngIf="!hasFeedback && onlyShowSuccessfulCompileStatus" [ngbTooltip]="resultTooltip" container="body">
-                    {{ resultString }}
-                </span>
-                <span *ngIf="!hasFeedback && !onlyShowSuccessfulCompileStatus" [ngbTooltip]="resultTooltip" container="body">
-                    {{ result.resultString }}
-                </span>
+                <ng-container *ngIf="!resultIsPreliminary(result)">
```

Comment:
I mean function calls in html are to be omitted, but I think in this context the resulting html would become kinda ugly, so I guess its ok here